Makefile

```diff
@@ -83,12 +83,12 @@ canonize: manifests generate fmt vet envtest ## Canonize tests.
 helm-kind-install: ## Install helm chart from sources in kind.
 	docker build -t ${OPERATOR_IMAGE}:${OPERATOR_TAG} .
 	kind load docker-image ${OPERATOR_IMAGE}:${OPERATOR_TAG}
-	helm install ytsaurus $(OPERATOR_CHART)
+	helm upgrade -i ytsaurus $(OPERATOR_CHART) --set controllerManager.manager.image.repository=${OPERATOR_IMAGE} --set controllerManager.manager.image.tag=${OPERATOR_TAG}
```

Comment:
could explain the change (maybe add comment)

Reply:
It's useful for local testing. Let's say you don't want to recreate kind cluster, but would like to upgrade operator's image. In that case you have to specify a new OPERATOR_TAG in Makefile and run `make helm-kind-install`. Operator's image will be upgraded, using helm

Reply:
You mean to use some prebuilt operator image, not the one we building from source?

Reply:
Maybe better to get OPERATOR_TAG from envvar with current default so you don't need to edit Makefile for that action